flink-libraries/flink-table/src/test/scala/org/apache/flink/api/scala/batch/table/AggregationsITCase.scala

```diff
@@ -400,5 +400,37 @@ class AggregationsITCase(
     TestBaseUtils.compareResultAsText(results.asJava, expected)
   }
 
+  @Test
+  def testAggregateEmptyDataSets(): Unit = {
```

Comment:
Table API and SQL are internally represented the same way.
We can drop the expensive ITCase for the Table API and add tests that check the plan based on `TableTestBase` for the Table API instead.